FusionIIIT/templates/programme_curriculum/view_curriculums_of_a_programme.html

```diff
@@ -139,7 +139,7 @@
 {% block rightcontent %}
 <br>
 <br>
-<div class="ui center aligned segment">
+<!-- <div class="ui center aligned segment">
```

Comment:
Why you are commenting it out..? Just Remove the whole part..

Reply:
Done .